app/models/user.rb

```diff
@@ -102,13 +102,9 @@ def self.from_omniauth(access_token)
   # (e.g. assigned to current line and in user.patients)
 
   def recently_called_patients(lines = LINES)
-    patients.in(_line: lines)
-            .select { |patient| recently_called_by_user? patient }
-  end
-
-  def call_list_patients(lines = LINES)
-    patients.in(_line: lines)
-            .reject { |patient| recently_called_by_user? patient }
+    Patient.in(_line: lines, user_ids: BSON::ObjectId(id)).where( :$and =>
+    [ { :"calls.updated_at"  => (8.hours.ago..Time.now) },
```

Comment:
two things here: 

a) let's set 8.hours.ago to a constant of some kind, and swap this out. Makes it a little more readable imo
b) we'll want `Time.zone.now` rather than just `Time.now`. Time zones are, as always, actual hell on earth, but ruby's time objects has a lot of good ways around it.